<launch>
  <node pkg="jiwon_mission" type="leftright" name="leftright" output="screen">
    <param name="image_topic" value="/camera/image_rect_color" />
    <param name="left_template_path"  value="$(find jiwon_mission)/data/left.png" />
    <param name="right_template_path" value="$(find jiwon_mission)/data/right.png" />
    <param name="ratio_thresh" value="0.75" />
    <param name="min_good_matches" value="10" />
    <param name="use_circle_roi" value="true" />
  </node>
</launch>
